src/main/java/nz/cri/gns/NSHM/opensha/ruptures/NSHMRuptureSetBuilder.java

```diff
@@ -0,0 +1,265 @@
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.plausibility.PlausibilityConfiguration;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.plausibility.impl.CumulativeAzimuthChangeFilter;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.plausibility.impl.JumpAzimuthChangeFilter;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.plausibility.impl.MinSectsPerParentFilter;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.plausibility.impl.TotalAzimuthChangeFilter;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.strategies.ClusterConnectionStrategy;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.strategies.ClusterPermutationStrategy;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.strategies.ConnectionPointsPermutationStrategy;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.strategies.DistCutoffClosestSectClusterConnectionStrategy;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.strategies.UCERF3ClusterPermuationStrategy;
+import org.opensha.sha.earthquake.faultSysSolution.ruptures.util.SectionDistanceAzimuthCalculator;
+import org.opensha.sha.faultSurface.FaultSection;
+
+import com.google.common.base.Preconditions;
+
+import nz.cri.gns.NSHM.opensha.ruptures.downDipSubSectTest.DownDipSubSectBuilder;
+import nz.cri.gns.NSHM.opensha.ruptures.downDipSubSectTest.DownDipTestPermutationStrategy;
+import nz.cri.gns.NSHM.opensha.ruptures.downDipSubSectTest.SubSectionParentFilter;
+import nz.cri.gns.NSHM.opensha.util.FaultSectionList;
+import scratch.UCERF3.SlipAlongRuptureModelRupSet;
+import scratch.UCERF3.enumTreeBranches.FaultModels;
+import scratch.UCERF3.enumTreeBranches.ScalingRelationships;
+import scratch.UCERF3.enumTreeBranches.SlipAlongRuptureModels;
+
+/**
+ * Builds opensha SlipAlongRuptureModelRupSet rupture sets 
+ * using NZ NSHM configurations for:
+ *  - plausability
+ *  - rupture permutation Strategy (with different strategies available for test purposes)
+ */
+public class NSHMRuptureSetBuilder {
+
+	static DownDipSubSectBuilder downDipBuilder;	
+	Set<Integer> faultIdIn = Collections.emptySet();
+	
+	double maxSubSectionLength = 0.5; 	// maximum sub section length (in units of DDW)
+	double maxDistance = 5; 			// max distance for linking multi fault ruptures, km
+	long maxFaultSections = 100000; 	// maximum fault ruptures to process
+	long skipFaultSections = 0; 		// skip n fault ruptures, default 0"
+	int numThreads = Runtime.getRuntime().availableProcessors(); // use all available processors
+	int minSubSectsPerParent = 2;		// 2 are required for UCERf3 azimuth calcs
+	RupturePermutationStrategy permutationStrategyClass = RupturePermutationStrategy.DOWNDIP;
+	
+	public enum RupturePermutationStrategy {
+		DOWNDIP, UCERF3, POINTS,
+	}
+	
+	/**
+	 * 
+	 * @return NSHMRuptureSetBuilder the builder, with the default NSHM configuration.
```

Comment:
A constructor should not have a `@return` tag. 

Should probably be something like "Constructs a new NSHMRuptureSetBuilder with the default NSHM configuration."